tests/common.go

```diff
@@ -4865,3 +4862,23 @@ func GetRandomStorageLessNode(slNodes []node.Node) node.Node {
 	}
 	return node.Node{}
 }
+
+func Waiter(task func() bool, taskDescription string, timeoutPeriod int, intervalPeriod int, timeUnit time.Duration) {
```

Comment:
I think you can use existing function task.DoRetryWithTimeout for waiting and checking if functon is successful

Reply:
Fixed it. Removed my Waiter function.